### ISSUE TITLE
Currently to easy to call shuttle.

### ISSUE THREAD

**Issue by RocKeD** [tgstation/-tg-station#9278]
Hello, currently I believe the shuttle is to easy to call. It seems most rounds end a lot earlier than they really need to. This might be because something bad happened like an explosion and then people freak out and call the shuttle, or it could be that the clown has all access and hes decided to call the shuttle early. My idea to solve these problems plus many more is that the number of id's required to call the shuttle should be increased by 1 or 2 meaning that two or three command id's have to be put in the machine before the shuttle may be called. This will have more benefits for the station as people must work together if a threat is at hand to get the shuttle called, no more one man army style (unless you have two or three command id's). Another benefit is that rounds will tend to last longer as a result of increased teamwork engineering for example will have more importance, it seems that is it rare nowadays to see the station ACTUALLY get repaired, I can't remember the last time I saw a whole room repaired machines and all. Tell me what you guys think of this idea, I was originally going to submit this to the forums first but apparently because I created a new account I have limited posting capabilities.
(Feel free to post this idea/suggestion on the forums, although I would like this copy/pasted or a url pointing to this post please.)
EDIT: Never mind, found out I had to link accounts to post in the forums.
         -https://tgstation13.org/phpBB/viewtopic.php?f=2&t=3520

TL;DR: Increase required id's to call shuttle to at least two maybe three. This increases quality of play overall.

**Comment by Dorsisdwarf** [comment]
The only problem with this is "law 2, call the shuttle AI"
And if you remove that, traitor AIs get duffed over.

**Comment by Jordie0608** [comment]
Discussion like this is suited for the forums, closing this
